docs/api.md

```diff
@@ -299,7 +300,7 @@ those common validations are defined here.
 * `HEX_STRING`: `/^(?:[a-fA-F0-9]{2})+$/`
 * `URLSAFEBASE64`: `/^[a-zA-Z0-9-_]*$/`
 * `BASE_36`: `/^[a-zA-Z0-9]*$/`
-* `URL_SAFE_BASE_64`: `/^[A-Za-z0-9_-]*$/`
+* `URL_SAFE_BASE_64`: `/^[A-Za-z0-9_-]+$/`
```

Comment:
> We should probably delete one or other and update the other references.

I can make a separate PR for that actually, no need to stink this one up with unrelated changes. (and thanks for updating the docs, guess I must have forgot at some point for this definition to be out-of-date)